apply-notch-hax.sh

```diff
@@ -36,19 +54,20 @@ for file in $(grep '^diff' $DIFF | grep -Eo '\ b/.*' | cut -c 4-); do
 	cp /$file $WORK/root/$file
 done
 
-cd root/
+pushd root/
 if ! patch -p1 < $DIFF; then
 	echo ">> ERROR: Some system files are incompatible with the patches;
           Please adjust '$DEVICE.diff' and try again!"
 	exit 1
 fi
-cd ../
+popd
 
 echo ">> Patches applied successfully! Proceeding to replacing system files..."
 mount | grep -q ' / .*ro' && sudo mount -o remount,rw /
 sudo cp -r root/* /
 sudo mount -o remount,ro /
 
+cd ..
```

Comment:
Meh, no need for this as `$WORK` itself isn't getting removed; might as well stay there for the rest of the script.